docs/recommendations/DNA/alleles.md

```diff
@@ -76,32 +89,32 @@ Allele: a series of variants on one chromosome.
     - disease severity will depend on the combination of variants found
     - in recessive disease, when two variants are on one allele an individual is a carrier or you might not have found the variant on the 2nd allele
 
-!!! note "I find the notation c.[76A>C] without describing the second allele misleading; not enough researchers know this refers to only one of the two alleles present. Would using c.[76A>C];[] be OK?"
+!!! note "I find the notation `c.[76A>C]` without describing the second allele misleading; not enough researchers know this refers to only one of the two alleles present. Would using `c.[76A>C];[]` be OK?"
 
-    No, the recommended description is LRG_199t1:c.[76A>C];[76=], i.e. c.76= for "no change" on the second allele.
+    No, the recommended description is `LRG_199t1:c.[76A>C];[76=]`, i.e. `c.76=` for "no change" on the second allele.
 
 !!! note "How should I describe the variants detected in males and females for a gene on the X-chromosome?"
 
-    In **females** the description is straightforward, like LRG_199t1:c.[76A>C];[76=]. In **males** there is no second allele (X-chromosome) which can be described as LRG_199t1:c.[76A>C];[0], i.e. using "**c.0**" to indicate the absence of a second X-chromosome.
+    In **females** the description is straightforward, like `LRG_199t1:c.[76A>C];[76=]`. In **males** there is no second allele (X-chromosome) which can be described as `LRG_199t1:c.[76A>C];[0]`, i.e. using `c.0` to indicate the absence of a second X-chromosome.
 
-!!! note "I have a patient with hearing loss and variants in the GJB2 (c.35delG) and GJB6 (c.689*690insT) genes, how should I describe this? *(Nancy Carson, Ottawa, Canada)\_"
+!!! note "I have a patient with hearing loss and variants in the GJB2 (`c.35delG`) and GJB6 (`c.689*690insT`) genes, how should I describe this? \*(Nancy Carson, Ottawa, Canada)\_"
 
-    The recommendation is to use the format GJB2:c.[35delG] GJB6:c.[689_690insT]. This uses standard HGVS descriptions and prevents confusion regarding which variant was found in which gene. Note it is essential that you also define the coding DNA reference sequence used. Another format, coping with this directly, is to describe the variants as NM_004004.2:c.[35delG] NM_006783.1:c.[689_690insT], i.e. using the Genbank reference sequences in stead of the HGNC approved Gene Symbol.
+    The recommendation is to use the format `GJB2:c.[35delG] GJB6:c.[689_690insT]``. This uses standard HGVS descriptions and prevents confusion regarding which variant was found in which gene. Note it is essential that you also define the coding DNA reference sequence used. Another format, coping with this directly, is to describe the variants as `NM_004004.2:c.[35delG] NM_006783.1:c.[689_690insT]`, i.e. using the Genbank reference sequences in stead of the HGNC approved Gene Symbol.
```

Comment:
I haven't looked in detail yet at the changes in the discussion section, but I noticed there's a backtick too many after the first variant description.